code/modules/mining/abandoned_crates.dm

```diff
@@ -7,14 +7,14 @@
 	icon_state = "securecrate"
 	var/code = null
 	var/lastattempt = null
+	var/lastattempt2 = null
```

Comment:
Except ```lastattempt``` literally isn't used anymore, you've completely removed it's usage, it's set to input and then never read, meaning it does nothing.